content/operate/kubernetes/reference/supported_k8s_distributions.md

```diff
@@ -12,6 +12,8 @@ weight: 30
 
 Each release of Redis Enterprise for Kubernetes is thoroughly tested against a set of Kubernetes distributions. The table below lists Redis Enterprise for Kubernetes versions and the Kubernetes distributions they support.
 
+{{<note>}}Testing is done with the default container runtime for each platform. Other container runtimes are not tested or certified.”{{</note>}}
```

Comment:
I would remain more vague here.
Just mention what we test, do not say everything else is not supported.